src/presentation/pages/Note.vue

```diff
@@ -54,6 +70,41 @@ const { note, noteTools, save, noteTitle, canEdit } = useNote({
   id: noteId,
 });
 
+const lastEdit = ref<string>('Last edit ');
+
+/**
+ * Create new child note
+ */
+function createChildNote(): void {
+  if (props.id === null) {
+    throw new Error('Note is Empty');
+  }
+  router.push(`/note/${props.id}/new`);
+}
+
+/**
+ * Move to note settings
+ */
+function redirectToNoteSettings(): void {
+  if (props.id === null) {
+    throw new Error('Note is Empty');
+  }
+  router.push(`/note/${props.id}/settings`);
+}
+
+/**
+ * Loads the time when the note was last modified when opened
+ */
+watch(note, () => {
+  if (note.value !== null && 'updatedAt' in note.value) {
+    const updatedAt = note.value.updatedAt;
+
+    if (typeof updatedAt === 'string') {
+      lastEdit.value = 'Last edit ' + formatShortDate(updatedAt);
+    }
+  }
+});
```

Comment:
Can we get rid of `lastEdit` variable and use `'Last edit ' + formatShortDate(note.value.updatedAt);` directly in template since note is reactive

Reply:
In this case we have propblem with types
![image](https://github.com/codex-team/notes.web/assets/94054053/01c9a6d1-ea5e-4d6a-bab9-9a18e04502af)

I think it's best to leave it as it is

Reply:
because you don't need to use `.value` in template

Reply:
![image](https://github.com/codex-team/notes.web/assets/94054053/d730ec1e-57cb-46f2-8a0a-788ddda2d8a3)

The result same without .value